README.md

```diff
@@ -325,7 +325,7 @@ The `configure` request object is composed as follows:
     - **`protocol`** - _`URI string`_: a URI that represents the protocol being configured.
     - **`types`** - _`object`_: an object that defines the records that can be used in the `structure` graph of the `definition` object. The following properties are optional constraints you can set for the type being defined:
       - **`schema`** - _`URI string`_ (_optional_): the URI of the schema under which the record will be bucketed.
-      - **`dataFormat`** - _`Media Type string`_ (_optional_): the IANA string corresponding with the format of the data the record will be bucketed. See IANA's Media Type list here: https://www.iana.org/assignments/media-types/media-types.xhtml
+      - **`dataFormats`** - _`Media Type string[]`_ (_optional_): Array of the IANA strings corresponding with the formats of the data the record will be bucketed. See IANA's Media Type list here: https://www.iana.org/assignments/media-types/media-types.xhtml
```

Comment:
@csuwildcat does "bucketed" make sense to use in the case of describing the purpose of `dataFormats` specified as a constraint on prototype types?